docs/components/contributors.vue

```diff
@@ -0,0 +1,266 @@
+          website: entry.fromAccount.website,
+          status: entry.status,
+          // For recurring donations, this is the installment amount
+          // For one time donations, this is the donation amount (most recent)
+          amount: entry.amount.value,
+          // For recurring donations, this is the total amount donated
+          // For users that donate multiple times, this will be the total of all one time donations
+          totalAmount: entry.totalDonations.value,
+          // For recurring donations, this is how often the donation is received
+          frequency: entry.frequency,
+          // We now have sponsor tiers, but some appear as
+          // `null` (they were made before the tiers were created)
+          tier: (entry.tier || {}).slug || null,
+          date: new Date(entry.createdAt)
+        }
+      })
+    },
+    sortCompare(a = {}, b = {}) {
+      // Sort first by amount, then by date
+      return (b.totalAmount || 0) - (a.totalAmount || 0) || (b.date || 0) - (a.date || 0)
+    },
+    processPlatinumSponsors(platinumSponsors = []) {
+      // Platinum sponsors are provided in reverse chronological order
+      // so we sort by larger amount first, then by date
+      this.platinumSponsors = platinumSponsors.sort(this.sortCompare)
+    },
+    processGoldSponsors(goldSponsors = []) {
+      // Gold sponsors are provided in reverse chronological order
+      // so we sort by larger amount first, then by date
+      this.goldSponsors = goldSponsors.sort(this.sortCompare)
+    },
+    processSilverSponsors(silverSponsors = []) {
+      // Silver sponsors are provided in reverse chronological order
+      // so we sort by larger amount first, then by date
+      this.silverSponsors = silverSponsors.sort(this.sortCompare)
+    },
+    processBronzeSponsors(bronzeSponsors = []) {
+      // Bronze sponsors are provided in reverse chronological order
+      // so we sort by larger amount first, then by date
+      this.bronzeSponsors = bronzeSponsors.sort(this.sortCompare)
+    },
+    processBackers(backers = []) {
+      // Backers are provided in reverse chronological order
+      // so we sort by larger amount first, then by date
+      // Limit to top 16 backers
+      this.backers = backers.sort(this.sortCompare).slice(0, MAX_BACKERS)
+    },
+    processDonors(donors = []) {
+      // Donors are provided in reverse chronological order,
+      // but donors can be listed more than once (for each individual donation),
+      // although the `totalDonations` is the same on each entry
+      // We sort by larger amount first, then by date
+      // Limit to top 32 most recent donors
+      this.donors = donors
+        .reduce((results, donor) => {
+          if (results.map(d => d.slug).indexOf(donor.slug) === -1) {
+            results.push(donor)
+          }
+          return results
+        }, [])
+        .sort(this.sortCompare)
+        .slice(0, MAX_DONORS)
+    },
+    visibleHandler() {
```

Comment:
This should check the first argument to the handler to make sure that it is `true` (it can be called initially with `false`)